src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,19 @@
 package core.basesyntax;
 
 public class Application {
+    private static final int NUMBER_OF_BALLS = 3;
+    private static final Lottery lottery1 = new Lottery();
+
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery[] lottery = new Lottery[NUMBER_OF_BALLS];
```

Comment:
An array of lottery may contain only lottery objects. I guess, you thought, that this array would contain balls. For this task, there's no reason to create more than one lottery.

Reply:
im lost. we need 3 balls. to create 1 ball we need to call one lottery, am I r?

Reply:
did it the same way I used it in the previous exercise